canvasapi/account.py

```diff
@@ -1688,6 +1689,24 @@ def import_outcome(self, attachment, **kwargs):
             if is_path:
                 attachment.close()
 
+    def query_by_account(self, **kwargs):
```

Comment:
Consider renaming to include `audit`, to make it more clear what this endpoint does relative to an `Account` object. Also update the description.

Do the same for `Course.query_by_course()`